recipe/meta.yaml

```diff
@@ -81,6 +100,28 @@ outputs:
       license: BSD-3-Clause
       license_file: LICENSE
 
+  - name: gxx
+    script: install-symlinks.sh
+    build:
+      skip: True  # [target_platform != ctng_target_platform]
+    requirements:
+      - gxx_impl_{{ target_platform }} {{ ctng_gcc }}.*
+      - gcc {{ ctng_gcc }}.*
```

Comment:
I am totally fine without the explicit `run` section here. I have just not encountered that before!

Reply:
Without the explicit run, it is both host and run. I want `gxx_impl` for both host and run

Reply:
oh man that is a fun shortcut. I wonder if the bot does that properly when it reads recipes.